.github/workflows/changelog.yml

```diff
@@ -0,0 +1,24 @@
+name: changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled, reopened]
```

Comment:
Is there anything we exclude here? Can't it just be `on: pull_request`?

Reply:
It needs to have labeled and unlabeled because if we add or remove "skip news" the pipeline result is not the same.